config/names/last.txt

```diff
@@ -616,4 +566,1061 @@ Mikhaylov
 Pavlov
 Semenov
 Andreev
-Alekseev
\ No newline at end of file
+Alekseev
+Mali
+Troshani
+Nano
+Kupi
+Syla
+Lucca
+Hasani
+Khalef
+Benita
+Fahas
+Seghir
+Ziani
+Saadi
+Beghal
+Yamin
+Mousa
+Bilal
+Khalaf
+Zafar
+Salman
+Naderi
+Usman
+Al-Hassen

```

Comment:
I _think_ this (being hyphens in names) might have problems because of byond and sanitizing. Could be wrong.

Reply:
Taj names sometimes have hyphens and it usually works out fine iirc. Sanitizing usually strips out things related to HTML like ``<`` or ``>`` and not ``-``.

Reply:
When testing they showed up fine, and yeah what Neerti said. It isn't a banned character.